packages/docs/src/content/docs/guides/performance.md

```diff
@@ -74,8 +74,8 @@ messages like this:
 ```
 
 The first number in `P1/1` is the number of the program, the second number
-indicates additional entry files were found in the previous round so it does
-another round of analysis on those files.
+indicates additional entry files were found so it does another round of analysis
```

Comment:
(not related to these changes) but I don't get this part

> The first number in `P1/1` is the number of the program

What is "the number of the program"? Is it an id representing an "instance" of Knip associated with a given `tsconfig.json`, or is it something else entirely?

Reply:
Hopefully answered here in the FAQ now: https://github.com/webpro-nl/knip/blob/8df0855f645e026e6e3e379ffb14c72147212afc/packages/docs/src/content/docs/reference/faq.md#why-doesnt-knip-match-my-typescript-project-structure

Another take on this, in short:

- workspace: dir with `package.json` (package manager concept)
- project: dir with `tsconfig.json` (typescript concept) - not relevant here
- program: Knip creates a typescript program and then adds as many workspaces as its config allows

Guess this is worth another entry in the FAQ, I'll keep it in mind.